apps/experiments/models.py

```diff
@@ -596,6 +691,21 @@ def is_public(self) -> bool:
     def is_participant_allowed(self, identifier: str):
         return identifier in self.participant_allowlist or self.team.members.filter(email=identifier).exists()
 
+    @transaction.atomic()
+    def delete(self, *args, **kwargs) -> tuple:
```

Comment:
I think I'm rather going to take this out, since [this PR](https://github.com/dimagi/open-chat-studio/pull/676) is in the pipeline. Archiving a single version is supported by that PR, and I suppose it wouldn't make much sense to have an archive button for the working experiment. If we delete it, everything related goes, which is fine.

Reply:
https://github.com/dimagi/open-chat-studio/pull/671/commits/a46bb96e3050ca97ae118beaa030654f0d883a96